src/System.Windows.Forms/src/System/Windows/Forms/RichTextBox.cs

```diff
@@ -3533,8 +3533,8 @@ private void WmSelectionChange(Richedit.SELCHANGE selChange)
             if (ImeMode == ImeMode.Hangul || ImeMode == ImeMode.HangulFull)
             {
                 // Is the IME CompositionWindow open?
-                int compMode = unchecked((int)(long)User32.SendMessageW(this, (User32.WM)RichEditMessages.EM_GETIMECOMPMODE));
-                if (RichTextBoxConstants.ICM_NOTOPEN != compMode)
+                ICM compMode = (ICM)unchecked((int)(long)User32.SendMessageW(this, (User32.WM)RichEditMessages.EM_GETIMECOMPMODE));
```

Comment:
I believe you can write as follows:
```suggestion
                ICM compMode = unchecked((ICM)(long)User32.SendMessageW(this, (User32.WM)RichEditMessages.EM_GETIMECOMPMODE));
```

We have few examples of such conversion in the codebase, e.g.:
https://github.com/dotnet/winforms/blob/af6d6c6e4f38eebc2cb64a0dc3fee6872539b0ef/src/System.Windows.Forms/src/System/Windows/Forms/Control.cs#L9547
https://github.com/dotnet/winforms/blob/af6d6c6e4f38eebc2cb64a0dc3fee6872539b0ef/src/System.Windows.Forms/src/System/Windows/Forms/ComboBox.cs#L3892